Formula/restic.rb

```diff
@@ -0,0 +1,28 @@
+class Restic < Formula
+  desc "Backups done right"
+  homepage "https://restic.github.io/"
+  url "https://github.com/restic/restic/archive/v0.7.3.tar.gz"
+  sha256 "6d795a5f052b3a8cb8e7571629da14f00e92035b7174eb20e32fd1440f68aaff"
+  head "https://github.com/restic/restic.git"
+
+  depends_on "go" => :build
+
+  def install
+    system "make"
+    bin.install "restic"
```

Comment:
Could you ask the developers to provide a `make install` target?

Reply:
No, using `make` is good, but it would be nice to be able to use it for installing the binary as well as compiling.

Reply:
i can ask, but since that's not really a `go` feature (binaries are supposed to be installed in `$GOBIN`), i doubt they want to do something that platform dependent and away from `go` conventions

Reply:
IMO, just use `go build` here. There's plenty of examples in existing formulae if you need 'em. I don't think we're gaining anything appreciable by running `make` but ILZ/Alex/etc can overrule me.

Reply:
gotcha, since there seems to be consensus on using the `go` build system, i'll switch to that